libs/seiscomp/core/recordsequence.h

```diff
@@ -175,6 +175,8 @@ class SC_SYSTEM_CORE_API TimeWindowBuffer : public RecordSequence {
 		//! Return percentage of available data within TimeWindowBuffer's TimeWindow
 		double availability() const;
 
+		//! Return the TimeWindow used in constructing the TimeWindowBuffer
+		const TimeWindow& timeWindow() const;
```

Comment:
Please replace `const TimeWindow& timeWindow() const` with `const TimeWindow &timeWindow() const` for consistency.